app/presenters/flood_risk_engine/enrollment_presenter.rb

```diff
@@ -38,6 +38,11 @@ def correspondence_contact_name
       title
     end
 
+    def organisation_registration_number
+      return unless enrollment.organisation && enrollment.organisation.registration_number.present?
+      enrollment.organisation.registration_number
+    end
+
     private
```

Comment:
I think you can just add to the the delagate call at the top of the class - will handle nil, e.g. 
```
delegate :name,
             :primary_address,      
             :registration_number,
             to: :organisation, allow_nil: true, prefix: true
```